tests/cache/tests.py

```diff
@@ -1309,6 +1315,13 @@ def test_pylibmc_legacy_options(self):
         self.assertIsInstance(warns[0].message, RemovedInDjango21Warning)
         self.assertEqual(str(warns[0].message), deprecation_message)
 
+    def test_close(self):
+        # Override base test, since libmemcached manages it's own connections, and thus
+        # PyLibMCCache's close() is intentionally a no-op (#11331).
+        with mock.patch.object(cache._lib.Client, 'disconnect_all', autospec=True) as mock_disconnect:
+            signals.request_finished.send(self.__class__)
+        self.assertFalse(mock_disconnect.called)
```

Comment:
Perhaps it would be useful to add a class attribute on the test class like `should_disconnect_on_close = True` and use that in the assertion, `self.assertIs(mock_disconnect.called, self.should_disconnect_on_close)` so as not to repeat the test.